gce/image/build_image.sh

```diff
@@ -18,7 +18,6 @@ DRY_RUN=false
 source ../../SCYLLA-VERSION-GEN
 
 PRODUCT=$(cat build/SCYLLA-PRODUCT-FILE)
-BUILD_ID=$(date -u '+%FT%H-%M-%S')
```

Comment:
if you delete the build_id how it will be passed in line 195?

Reply:
we provide it as a parameter

Reply:
I can set maybe a default value like - `000` or something

Reply:
let's keep it